js/main.js

```diff
@@ -79,6 +79,20 @@ function harvests(cropID) {
 	var crop = seasons[options.season].crops[cropID];
 	var fertilizer = fertilizers[options.fertilizer];
 
+	// if the crop is cross season, add 28 extra days for each extra season
+	var remainingDays = options.days;
+	if (options.crossSeason && options.season < 2) {
+		for (var i = options.season + 1; i < 3; i++) {
+			for (var j = 0; j < seasons[i].crops.length; j++) {
```

Comment:
I'd usually cache this in a dictionary, but the arrays are very small - what makes the O(m*n) unnoticeable. Caching it would just made the code more complicated without a good reason.